src/release_notes_workflow/README.md

```diff
@@ -0,0 +1,29 @@
+#### Components Release Notes Check 
+
+Pulls the latest CommitID based on user passed argument `--date` and checks if the release notes exists or not. Outputs a formated markdown table as follows.
```

Comment:
This text is incorrect. It pulls the latest code, then checks whether release notes have been written and whether new commits have been made since date.